### PR TITLE
Fix exception thrown using MishActivation

### LINKED ISSUE #51: Torch 1.9.0 does not have torch.nn.functional.mish
I have pulled the 11.3 version docker image from the README.md, which has torch 1.9.0 installed. However, the code could not be run with torch 1.9.0 since it does not have the function "mish". Is there anything we can do about this?

### PR DESCRIPTION
**Pull request by chinoll**
# What does this PR do?
Fix an exception when calling MishActivation module when using pytorch 1.9.0

Fixes [#42](https://github.com/microsoft/GLIP/issues/51)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by sgugger** [comment]
What exception are we fixing? The [PyTorch documentation](https://pytorch.org/docs/1.9.0/generated/torch.nn.functional.mish.html?highlight=mish#torch.nn.functional.mish) indicates the mish function was in 1.9.0 and I just tried on a fresh env, the function can be imported without any problem on 1.9.0.

**Comment by chinoll** [comment]
> What exception are we fixing? The [PyTorch documentation](https://pytorch.org/docs/1.9.0/generated/torch.nn.functional.mish.html?highlight=mish#torch.nn.functional.mish) indicates the mish function was in 1.9.0 and I just tried on a fresh env, the function can be imported without any problem on 1.9.0.
I am using pytorch version 1.9.0a0+c3d40fd, and using nn.functional.mish will throw AttributeError exception.

File: src/transformers/activations.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ class MishActivation(nn.Module):
 
     def __init__(self):
         super().__init__()
-        if version.parse(version.parse(torch.__version__).base_version) < version.parse("1.9"):
+        if version.parse(version.parse(torch.__version__).base_version) < version.parse("1.9.1"):
```
Comment (sgugger) [inline src/transformers/activations.py:124]:
```suggestion
        if version.parse(version.parse(torch.__version__).base_version) < version.parse("1.9.0"):
```

**Review by sgugger — COMMENTED** [review]
If you want to exclude the development versions prior to the official 1.9.0, this is the right way (the test returns True for `"1.9.0a0+c3d40fd"`, just tried locally)

File: src/transformers/activations.py (hunk 1 above)
Comment (chinoll) [reply src/transformers/activations.py:124]: It seems to have a different result in my env, it returns False
image

Comment (sgugger) [reply src/transformers/activations.py:124]: Sorry I removed the base version in my case:
```
version.parse(torch.__version__) < version.parse("1.9.0")
```

**Review by sgugger — APPROVED** [review]
Thanks for iterating!

**Pull request merged**